package.json

```diff
@@ -39,10 +39,9 @@
     "test.watch": "yarn test --watch"
   },
   "dependencies": {
-    "@stoplight/types": "^11.0.0",
-    "@types/js-yaml": "3.12.1",
+    "@stoplight/types": "^11.1.0",
     "lodash": "^4.17.15",
-    "yaml-ast-parser": "~0.0.43"
+    "yaml-ast-parser": "stoplightio/yaml-ast-parser#ba8043e2eb7b444789a424754935f991572c25a1"
```

Comment:
uh... shouldn't we publish it?

Reply:
@marbemac thoughts?

Reply:
Yup, done here - https://www.npmjs.com/package/@stoplight/yaml-ast-parser?activeTab=versions.